libraries/APM_Control/AR_AttitudeControl.cpp

```diff
@@ -239,6 +246,37 @@ float AR_AttitudeControl::get_steering_out_rate(float desired_rate, bool skid_st
     return constrain_float(p + i + d, -1.0f, 1.0f);
 }
 
+// get latest desired turn rate in rad/sec (recorded during calls to get_steering_out_rate)
+float AR_AttitudeControl::get_desired_turn_rate() const
+{
+    // return zero if no recent calls to turn rate controller
+    if ((_steer_turn_last_ms == 0) || ((AP_HAL::millis() - _steer_turn_last_ms) > AR_ATTCONTROL_TIMEOUT)) {
+        return 0.0f;
+    }
+    return _desired_turn_rate;
+}
+
+// get latest desired lateral acceleration in m/s/s (recorded during calls to get_steering_out_lat_accel)
+float AR_AttitudeControl::get_desired_lat_accel() const
+{
+    // return zero if no recent calls to lateral acceleartion controller
+    if ((_steer_lat_accel_last_ms == 0) || ((AP_HAL::millis() - _steer_lat_accel_last_ms) > AR_ATTCONTROL_TIMEOUT)) {
+        return 0.0f;
+    }
+    return _desired_lat_accel;
+}
+
+// get actual lateral acceleration in m/s/s.  returns true on success
+bool AR_AttitudeControl::get_lat_accel(float &lat_accel) const
```

Comment:
This isn't really your actual lateral acceleration - that's what comes from our sensors :-)

Maybe `predicted_lat_accel`?

Does this actually match `ACC.AccY` in testing?

Reply:
The ACC.AccY would be a body-frame lateral acceleration which is not what the controllers use.  It's possible this value is wrong but then I think that means the input to our controllers is also wrong.  I'm going to leave that analysis as an out-of-scope issue to be investigated later.